.chloggen/julianocosta89_debugexporter-profiles.yaml

```diff
@@ -0,0 +1,25 @@
+# Use this changelog template to create an entry for release notes.
+
+# One of 'breaking', 'deprecation', 'new_component', 'enhancement', 'bug_fix'
+change_type: enhancement
+
+# The name of the component, or a single word describing the area of concern, (e.g. otlpreceiver)
+component: debugexporter
```

Comment:
This is a chore PR, and only modifies internal packages. Shouldn't wee keep the changelog entry for when the marshaler is used?

Reply:
Agreed, I think we can remove the changelog entry